ts/store/actions/services.ts

```diff
@@ -27,6 +32,11 @@ export const loadService = createAsyncAction(
   "SERVICE_LOAD_FAILURE"
 )<string, ServicePublic, string>();
 
+// Remove services passing a list of tuples with serviceId and organizationFiscalCode
+export const removeServiceTuples = createStandardAction("SERVICES_REMOVE")<
```

Comment:
make the action name and it's identifier similar (e.g. `removeServices` or `SERVICES_REMOVE_TUPLES`) - probably the former